### PR TITLE
feat: Update projects to compile for Visual Studio 2019

### PR DESCRIPTION
**Pull request by darcywong00**
This tries to document the Developer steps we found to compile Speech Analyzer with Visual Studio 2019.

* Notable, this reverts the xerces-c submodule from #6 and documents the pre-requisite of building the library
* Remove the DistFiles/Xerces/ artifact files (keep LICENSE and NOTICE)
* (updated) Installer project also updated

Some caveats:
* If the full solution builds fine, does that mean all the tests work with VS 2019?
* I won't be attempting steps for ~"Installer Creation" or~ "Updating version tags" in this PR

### CODE REVIEW

File: DEVELOPER.md [hunk 1]
```diff
@@ -6,45 +6,59 @@ Speech Analyzer is a 32-bit application.  Use (x86) installers where appropriate
...
+1. Copy the following files from xerces-c\Build\Win32\VC14\Release\ into the created lib\ directory:
+  * xerces-c_3_1.dll
+  * xerces-c_3.lib
+
+Make note of these files because you will also copy them later for the Speech Analyzer build.
+
+## Install Fonts
+Some of the following fonts in `Lib\Fonts\Musique` need to be installed for using the music features:
+* Musique Unicode.ttf
+* MusicqueU.ttf
```
Comment (megahirt) [inline DEVELOPER.md:39]: I would suggest that we link directly to these TTF fonts to make it easy to download/install.

```suggestion
The following fonts need to be installed before using the music features:
* `[Lib\Fonts\Musique\Musique Unicode.ttf](https://github.com/sillsdev/SpeechAnalyzer/raw/master/Lib/Fonts/Musique/Musique%20Unicode.ttf)`
* `[Lib\Fonts\Musique\MusicqueU.ttf](https://github.com/sillsdev/SpeechAnalyzer/raw/master/Lib/Fonts/Musique/MusiqueU.ttf)`
```

**Review by megahirt — APPROVED** [review]
Well done @darcywong00 !  Would be great to have a screenshot of VS 2019 with a "Build Succeeded" message in this PR for posterity. :)

**Comment by darcywong00** [comment]
I updated DEVELOPER.md with additional info I needed to build the project on my other machine.

> Would be great to have a screenshot of VS 2019 with a "Build Succeeded" message in this PR for posterity. :)

image

File: Install/SpeechAnalyzer.iss [hunk 2]
```diff
@@ -43,16 +43,16 @@ Source: "..\Release\SA_DSP.dll"; DestDir: "{app}"; Flags: ignoreversion
 Source: "..\Release\SA_ENU.dll"; DestDir: "{app}"; Flags: ignoreversion
 Source: "..\Release\SA_DEU.dll"; DestDir: "{app}"; Flags: ignoreversion
 Source: "..\Release\SAUtils.dll"; DestDir: "{app}"; Flags: ignoreversion
+Source: "..\Release\xerces-c_3_1.dll"; DestDir: "{app}"; Flags: ignoreversion
```
Comment (darcywong00) [inline Install/SpeechAnalyzer.iss:46]: From this PR, this dll is manually copied to the `Release\` and `Debug\` folders instead of from `DistFiles\`

**Pull request merged**